packages/tools/devtools/devtools-browser-extension/package.json

```diff
@@ -13,7 +13,7 @@
 	"author": "Microsoft and contributors",
 	"scripts": {
 		"build": "fluid-build . --task build",
-		"build-and-test": "npm run build && npm run test",
+		"build-and-test": "npm run build && npm run test:e2e",
```

Comment:
Then should I keep the mocha test and jest test together by `npm run build && npm run test && npm run test:jest` or make another test script maybe `build:jest-test`?

Reply:
I'd say we don't need a build-and-jest-test version of this one. Jest tests can be their own script, and we'd do it in two steps, `npm run build` followed by `npm run test:jest`.

Reply:
Seems like the pipeline is failing cuz the rootDir(/FluidFramework) will run all jest:test scripts and /devtool-browser-extension doesn't have jest config yet. Can I change back to `e2e` and include this change in PR: https://github.com/microsoft/FluidFramework/pull/16142?

Reply:
Ok. Seems like we do need a script to start the test application and it will be included in jest.config file to kick out the server before jest test cases runs...